tests/regression/tests/REQUEST-930-APPLICATION-ATTACK-LFI/930100.yaml

```
@@ -51,3 +51,19 @@ tests:
             uri: "/?foo=0x2e.%000x2f0x2e.%00/WINDOWS/win.ini"
           output:
             log_contains: id "930100"
+  - test_title: 930100-4
+    desc: "Path Traversal Attack (<backslash>../), partially encoded"
```

```suggestion
    desc: "Status Page Test - Path Traversal Attack (<backslash>../), partially encoded"
```

Change to follow the convention we started in Varese.